app/views/locations/edit.html.erb

```diff
@@ -7,13 +7,13 @@
   ) %>
 <% end %>
 
-<%= render partial: "organisations/headings", locals: { main: "Add a location to this scheme", sub: @scheme.service_name } %>
-
 <%= form_for(@location, method: :patch, url: location_path) do |f| %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <%= f.govuk_error_summary %>
 
+<%= render partial: "organisations/headings", locals: { main: "Add a location to this scheme", sub: @scheme.service_name } %>
```

Comment:
<img width="975" alt="Screenshot 2022-08-03 at 10 27 53" src="https://user-images.githubusercontent.com/92022120/182574732-16712e69-65ba-43a3-9146-8684b7ab29dc.png">

Reply:
Think this should be indented 1 level in since it's inside the form tag?